src/main/java/edu/cornell/kfs/concur/businessobjects/ValidationResult.java

```diff
@@ -0,0 +1,52 @@
+package edu.cornell.kfs.concur.businessobjects;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ValidationResult {
+
+    protected boolean valid;
+    protected List<String> messages;
+
+    public ValidationResult(boolean valid, List<String> messages) {
+        this.valid = valid;
+        this.messages = messages;
+    }
+
+    public boolean isValid() {
+        return valid;
+    }
+
+    public void setValid(boolean valid) {
+        this.valid = valid;
+    }
+
+    public List<String> getMessages() {
+        return messages;
+    }
+
+    public void setMessages(List<String> messages) {
+        this.messages = messages;
+    }
+
+    public void addMessage(String message) {
+        if (messages == null) {
+            messages = new ArrayList<String>();
+        }
+        messages.add(message);
+    }
+
+    @Override
+    public String toString() {
+        StringBuffer result = new StringBuffer();
```

Comment:
Ok, maybe I'm just confused. Usually toString, would output the string representation of the object for things like logging, in which case I would expect the value of valid to be included somehow. I understand if no message is specified when the account info is valid, but how does that relate to the implementation of toString? Is toString being used somewhere in the logic to display/send the ValidationResults messages?